JamfSync/UI/FileSharePasswordView.swift

```diff
@@ -17,7 +17,7 @@ struct FileSharePasswordView: View {
         VStack(spacing: 10) {
             Text("File Share Password")
                 .font(.title)
-            Text("Enter the password for \(fileShareDp?.selectionName() ?? "")")
+            Text("Enter the password for \(fileShareDp?.selectionName() ?? ""): \(fileShareDp?.address ?? "")")
```

Comment:
Not gonna hold up the PR for it but not a huge fan of multiple ternaries in one line

Reply:
This one is pretty straight forward